packages/agent/src/parsers.ts

```diff
@@ -259,12 +261,21 @@ export async function receiveOutOfBandInvitation(
     )
   )
 
-  const { connectionRecord, outOfBandRecord } = await agent.oob.receiveInvitation(invitation, {
-    reuseConnection: true,
-  })
+  let connectionRecord: ConnectionRecord | undefined, outOfBandRecord: OutOfBandRecord
+
+  try {
+    ;({ connectionRecord, outOfBandRecord } = await agent.oob.receiveInvitation(invitation, {
+      reuseConnection: true,
+    }))
```

Comment:
This is not really cleaner, but I really don't like the weird syntax when you destructure and reasign let values

```suggestion
  try {
    const receiveInvitationResult = await agent.oob.receiveInvitation(invitation, {
      reuseConnection: true,
    })
    connectionRecord = receiveInvitationResult.connectionRecord
    outOfBandRecord = receiveInvitationResult.outOfBandRecord
```